tslint.json

```diff
@@ -78,5 +78,8 @@
     "rulesDirectory": [
         "node_modules/tslint-microsoft-contrib",
         "node_modules/tslint-eslint-rules/dist/rules"
-    ]
+    ],
+    "compilerOptions": {
```

Comment:
why need to change this file?

Reply:
It was an attempt to fix an issue with paths not being recognized for test cases. It was committed by accident